src/libs/installer/installer.pro

```diff
@@ -14,7 +14,8 @@ include(../../../installerfw.pri)
 #   SOURCES += $$PWD/productkeycheck.cpp
 #   ...
 #   your files if needed
-HEADERS += productkeycheck.h
+HEADERS += productkeycheck.h \
+    eve_public/app/platform.pb.h
```

Comment:
Why not declare this in line 52 instead (same place as the eve_launcher headers)?